src/common/base-class/construct-name-enum.ts

```diff
@@ -16,4 +16,5 @@ export enum ConstructName {
   CUSTOMSAGEMAKERENDPOINT = 'C5',
   HUGGINGFACESAGEMAKERENDPOINT = 'C6',
   JUMPSTARTSAGEMAKERENDPOINT = 'C7',
+  LANGFUSEDEPLOYMENT = 'C8',  // TODO: Pending confirmation
```

Comment:
```suggestion
  LANGFUSEDEPLOYMENT = 'C11',
```

Reply:
Fixed in af673346a3a04d2cb330602ee58b1ae0807f609f